css/index.css

```diff
@@ -131,45 +135,133 @@ header .logo {
     color: white;
 }
 
-/* Main Content Styles */
+.cta img {
+    max-width: 50%;
+} 
 
 .main-content {
    border-top: 2px solid black; 
    border-bottom: 2px solid black;
-   margin: 75px 0 35px 0;
-   padding: 35px 0;
-   font-size: 16px;
-   line-height: 18px;
-}
-
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
 }
 
 .main-content .middle-img {
+    width: 100%;
     display: block;
-    margin: 30px auto;
 }
 
+.main-content .top-content,
 .main-content .bottom-content {
     display: flex;
-    justify-content: space-around;
 }
 
-.main-content .bottom-content p {
-    padding-right: 20px;
+.main-content, .contact {
+    margin: 0 7%;
 }
 
-/* Contact Styles */
+.top-content, .bottom-content,
+.contact {
+    margin-top: 40px;
+}
 
-.contact p {
-    margin: 15px 0;
+.top-content, .bottom-content {
+    margin-bottom: 50px;
 }
 
-/* Footer Styles */
+.contact h4 {
+    margin-bottom: 15px;
+}
+
+.contact p {
+    margin-bottom: 20px;
+}
 
 footer {
     text-align: center;
-    margin: 40px 0 20px;
-}
\ No newline at end of file
+    margin-top: 40px;
+}
+
+@media (max-width: 600px) {
```

Comment:
Nice work with the break-points. You had brought up a key point where there were some weird funk going on as you resized the browser. It definitely looks a lot more fluid.